validator_client/src/preparation_service.rs

```diff
@@ -477,8 +477,8 @@ impl<T: SlotClock + 'static, E: EthSpec> PreparationService<T, E> {
             for batch in signed.chunks(self.validator_registration_batch_size) {
                 match self
                     .beacon_nodes
-                    .first_success(
-                        RequireSynced::No,
+                    .broadcast(
+                        RequireSynced::Yes,
```

Comment:
> Beacon nodes read validator data from the chain, and only send registration for active validators. However if the node is out of sync, it could either be missing the validator or have an outdated status of the validator, resulting in registration for the active validator to be missed.

Why is it best to require synced status? Eagerly sending registrations before synced status should not be harmful. If there any downside to it? Registrations will be sent every epoch regardless

Reply:
Yeah you're right - it only needs to require synced status if we're using `first_success` (as the node may exclude validators not active in local view; and in fact `RequireSynced::No` may miss registrations for some BNs connected to different relays, so this should be `Yes` here.

I'll fix this. Thanks!

Reply:
Fixed!